buildSrc/build.gradle

```diff
@@ -52,6 +52,19 @@ version = publishVersion
 def siteUrl = 'https://github.com/uber/okbuck'
 def gitUrl = 'https://github.com/uber/okbuck.git'
 
+def getReleaseRepositoryUrl() {
+  return hasProperty('RELEASE_REPOSITORY_URL') ? RELEASE_REPOSITORY_URL
+      : "https://oss.sonatype.org/service/local/staging/deploy/maven2/"
```

Comment:
We do not want to deploy snapshots here. We already have bintray which picks up our changes. If you want it purely for internal use, feel free to add this in a separate gradle script and use it via an apply block.